src/bikeracks/schema.sql

```diff
@@ -90,13 +87,5 @@ CREATE TABLE votes_history (
 /*want to be able to look up votes from a certain period in time */
 CREATE UNIQUE INDEX old_votes ON votes_history (vote_timestamp);
 
-/* TODO Things to check:
```

Comment:
These would be better as Issues.

Reply:
the TODO list? I think I deleted that comment. I mean, I don't see it anymore.

Reply:
I just mean in general. If you're tempted to write a TODO that's going to live long enough to get merged, it should have an issue associated with it.